flight/states/offaxis_odlc.py

```diff
@@ -0,0 +1,36 @@
+"""State to fly towards Off Axis ODLC for image capture"""
+import logging
+from mavsdk import System
+from ..state_settings import StateSettings
+from state import State
+from states.land import Land
+
+
+class OffAxisODLC(State):
+    """
+    State that handles Off Axis ODLC by flying to closest location w/in boundary and capture image
+
+    Attributes
+    ----------
+        N/A
+
+    Methods
+    -------
+        run() -> Land
+            Flies to closest location within flight boundary and record off-axis ODLC
+    """
+    async def run(self, drone: System) -> Land:
+        """
+        Flies to closest point within flight boundary and record image of Off-Axis ODLC
+
+        Parameters
+        ----------
+            drone: System
+                MAVSDK object for drone control
+
+        Returns
+        -------
+            Land : State
```

Comment:
Should just be `Land` not `Land : State`. 
The name to the left of the colon is supposed to refer to what you called that return variable name inside the function or some usefully named variable name that makes the return value more understandable